CASSANDRA-17881

nonstopdtop opened the pull request:
patch by Chris Thornett, Diogenese Topper; reviewed by -- for CASSANDRA-17881

Co-authored by: Chris Thornett <chris@constantia.io>
Co-authored by: Diogenese Topper <diogenese@constantia.io>

nonstopdtop committed:
CASSANDRA-17881

patch by Chris Thornett, Diogenese Topper; reviewed by -- for CASSANDRA-17881

Co-authored by: Chris Thornett <chris@constantia.io>
Co-authored by: Diogenese Topper <diogenese@constantia.io>

ErickRamirezAU reviewed (changes requested):
The images and captions won't render when I stage the page locally. I've spent 2 hours trying to get it to work but I've given up.

Please fix and test locally before submitting for review. 🙏

nonstopdtop committed:
image format fix

michaelsembwever commented:
<img width="1115" alt="Screenshot 2022-09-30 at 13 14 07" src="https://user-images.githubusercontent.com/559444/193258033-22f87194-8dc0-4662-a037-2bdb0d27fdc1.png">

--

<img width="1280" alt="Screenshot 2022-09-30 at 13 15 07" src="https://user-images.githubusercontent.com/559444/193258141-f5b10f10-371d-43ea-977c-5a643337b6cc.png">

--

<img width="1280" alt="Screenshot 2022-09-30 at 13 15 29" src="https://user-images.githubusercontent.com/559444/193258186-6f6d64bd-b8c3-4d3b-a1e6-dbd12f88768a.png">


--

<img width="1280" alt="Screenshot 2022-09-30 at 13 15 48" src="https://user-images.githubusercontent.com/559444/193258220-29755792-fbb0-4537-bd30-f78ecfac303c.png">

--
<img width="1280" alt="Screenshot 2022-09-30 at 13 16 07" src="https://user-images.githubusercontent.com/559444/193258275-d29c3e81-0a64-436c-b67a-edc737f5e765.png">

--
<img width="1280" alt="Screenshot 2022-09-30 at 13 16 28" src="https://user-images.githubusercontent.com/559444/193258319-cafa080f-91d2-4dd1-8e99-3cbd13e638af.png">

--

<img width="1280" alt="Screenshot 2022-09-30 at 13 16 41" src="https://user-images.githubusercontent.com/559444/193258349-f25ff561-13dd-46ab-a993-06a565e8d2be.png">


--
<img width="1280" alt="Screenshot 2022-09-30 at 13 16 57" src="https://user-images.githubusercontent.com/559444/193258401-1ec990cd-dcf7-4786-b4d3-cd55133c7021.png">


--
<img width="1280" alt="Screenshot 2022-09-30 at 13 17 12" src="https://user-images.githubusercontent.com/559444/193258435-32daff06-fd69-433a-bd25-bed82185d735.png">

michaelsembwever commented:
@ErickRamirezAU is it good now?

michaelsembwever commented:
@dtopdontstop the gracehopper image still looks busted

cthornett committed:
Update Apache-Cassandra-Changelog-19-September-2022.adoc

Both the Grace Hopper and Berlin Cassandra event changed to past tense and included reference to the London and Amsterdam city events.

cthornett committed:
Date change and image fix

added '/blog' to line 68: image::blog/gracehopperOSD.png[] to see if it fixes broken image. Changed publication date to 27 September to reflect changes in the text to past tense.

michaelsembwever commented on site-content/source/modules/ROOT/pages/blog.adoc:
```diff
@@ -8,6 +8,31 @@ NOTES FOR CONTENT CREATORS
+[discrete]
+=== Apache Cassandra Changelog #19
+[discrete]
+==== September 9, 2022
```
fixing this on the fly

michaelsembwever commented:
merged with 2af01b8f12fe3358f0c5193839b76ebe552ac3d6
